### DISCUSSION TITLE
TCA crash course

### DISCUSSION THREAD

**Discussion by bitsarat** [pointfreeco/swift-composable-architecture#2708] [Q&A] [answered]
Hi all, 
I was wondering if [A tour of the Composable Architecture 1.0](https://www.pointfree.co/collections/composable-architecture/composable-architecture-1-0) is a good way to get a fast intro to TCA? or do you really have to start from the beginning [here](https://www.pointfree.co/collections/composable-architecture) to understand all the concepts? Thanks in advance for helping.

**Comment by mbrandonw** [comment, marked as answer]
Hi @bitsarat, yes the 1.0 tour is the easiest way to get up to speed with the basics of the library, as well as the documentation in the repo. The full collection of episodes is more of a historic archive of all the thinking that went into building the library from the very beginning. It can be useful to understand why some decisions were made, or if you wanted to understand the nitty gritty details of the internals of the library.
